app/models/miq_group.rb

```diff
@@ -15,11 +15,11 @@ class MiqGroup < ApplicationRecord
   has_many   :miq_widget_sets, :as => :owner, :dependent => :destroy
   has_many   :miq_product_features, :through => :miq_user_role
 
-  virtual_column :miq_user_role_name, :type => :string,  :uses => :miq_user_role
+  virtual_delegate :name, :to => :miq_user_role, :allow_nil => true, :prefix => true
```

Comment:
How come you changed this to just `:name`? It seems ambiguous as it can be misconstrued as the name of the group when it's really the name of the groups role.

Reply:
This did not change

The target is `Role#name` has `prefix => true`, so the attribute name will be called `miq_user_role_name`